src/Web/Avalonia.Web.Blazor/AvaloniaView.razor.cs

```diff
@@ -254,9 +255,11 @@ protected override void OnAfterRender(bool firstRender)
                 Threading.Dispatcher.UIThread.Post(async () =>
                 {
                     _inputHelper = await InputHelperInterop.ImportAsync(Js, _inputElement);
+                    _canvasHelper = await InputHelperInterop.ImportAsync(Js, _htmlCanvas);
```

Comment:
It's required because setting cursor to the _inputElement is not working - because it's always display:none;